scaffolder-templates/build/.gitlab-ci.yml

```diff
@@ -0,0 +1,33 @@
+stages:
+  - wait
+  - update
+
+wait_for_value:
```

Comment:
this job is just to wait and make sure the variable is available?

Is it really needed? Is the context reloaded between each job of the same pipeline? If yes, a comment would be nice

Reply:
I believe there was a context reload as delaying the start of the update job was enough for the variables to register. The pipeline found a runner in the same speed in earlier checks w/o the sleep step, so I presume it was able to access the now-populated vars when the update step was starting.